src/pages/order-item/tabs/add-product-form/add-product-form.js

```diff
@@ -202,7 +201,6 @@ const AddProductForm = ({
             <TextField
               {...params}
               label={productLabels.product}
-              error={isFieldError(inputName.items)}
```

Comment:
you need this error, also you should add prop 'helperText' with the text of validation error
when the textfield has a valid value the error should disappear